### PR TITLE
Add sample to list connected cameras

### PR DESCRIPTION
**Pull request by torbsorb**
List connected cameras and print version information.

### CODE REVIEW

File: source/ZividNETSamples.sln [hunk 1]
```diff
@@ -122,6 +128,7 @@ Global
         HideSolutionNode = FALSE
     EndGlobalSection
     GlobalSection(ExtensibilityGlobals) = postSolution
-        SolutionGuid = {FD0FDA0F-31E4-4397-822E-0F1A9A4AED50}
+                SolutionGuid = {FD0FDA0F-31E4-4397-822E-0F1A9A4AED50}
+        SolutionGuid = {90099322-D7FF-42CF-A1D4-4DA13FA774B2}
```
Comment (SatjaSivcev) [inline source/ZividNETSamples.sln:132]: Why this change?

Comment (torbsorb) [reply source/ZividNETSamples.sln:132]: tab --> 4 spaces

Comment (torbsorb) [reply source/ZividNETSamples.sln]: reduced indentation to what it was, but now with spaces

Comment (SatjaSivcev) [reply source/ZividNETSamples.sln]: I don't think adding `SolutionGuid = {90099322-D7FF-42CF-A1D4-4DA13FA774B2}` is needed at all.

**Review by SatjaSivcev — APPROVED** [review]
LGTM
